src/test/java/math/FractionTest.java

```diff
@@ -88,8 +88,12 @@ public void testMultiplyFraction() {
 
     @Test
     public void testReciprocal() {
-        // TODO (N1SBRH)
-        fail();
+        assertFraction(5, 3, new Fraction(3,5).reciprocal());
+        assertFraction(2, 5, new Fraction(5,2).reciprocal());
+        assertFraction(15, 16, new Fraction(16,15).reciprocal());
+        assertFraction(-5, 3, new Fraction(-3,5).reciprocal());
+        ArithmeticException ex = assertThrows(ArithmeticException.class, () -> new Fraction(11, 0));
```

Comment:
This has nothing to do with testing the `reciprocal()` method, that method is not used here at all. Instead, you are testing the constructor here.